### PR TITLE
Add GoogleCalendar module

### PR DESCRIPTION
**Pull request by facetoe**
Add simple Google Calendar module.

### CODE REVIEW

**Comment by teto** [comment]
Just curious, could that work or be adapted to work with an alternative caldav server or is the API exclusive to GoogleCalendar ?

**Comment by facetoe** [comment]
@teto  currently it's exclusive to GoogleCalendar but I'm sure we could make it more generic.

**Comment by facetoe** [comment]
@teto Looks promising - https://developers.google.com/google-apps/calendar/caldav/v2/guide

**Comment by facetoe** [comment]
I'm going to close this PR and look into implementing this module using caldav.

**Comment by teto** [comment]
Cool !

2016-02-10 15:55 GMT+01:00 facetoe <notifications@github.com>:

**Comment by facetoe** [comment]
Sorry @teto, looks like a pain to impement caldav with Google Calendar. It might be worth making a separate module for caldav calendars.

File: dev-requirements.txt [hunk 1]
```diff
@@ -3,3 +3,5 @@ sphinx>=1.1
 colour>=0.0.5
 mock>=1.0
 pep8>=1.5.7
+google-api-python-client>=1.4.2
+python-dateutil>=2.4.2
```
Comment (enkore) [inline dev-requirements.txt]: These two should go into MOCK_MODULES in docs/conf.py: the dev-requirements should  only be used for packages needed to run the tests etc.

Comment (teto) [inline dev-requirements.txt]: No pb, I have no right to critisize ^^ thanks for considering it :)

2016-02-14 11:21 GMT+01:00 enkore <notifications@github.com>:

**Pull request merged**